### PR TITLE
Pagina de registro

### PR DESCRIPTION
**Pull request by JosephSC0121**
# Page Register 

## Description
Creacion pagina de registro
### Summary
image

## How Has This Been Tested?
Se testeo directamente en la web

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/modules/auth/templates/signup.tsx [hunk 1]
```diff
@@ -1,10 +1,77 @@
 "use client";
 
+import { useFormState } from "react-dom";
+import { authenticate } from "../services/actions";
+import { usePathname } from "next/navigation";
+import { Input } from "@/modules/common/components/input";
+import Link from "next/link";
+import { SubmitButton } from "../../common/components/submit-button";
+
```
Comment (Camilo-Camargo) [inline src/modules/auth/templates/signup.tsx:9]:
```suggestion
```

File: src/modules/auth/templates/signup.tsx
Comment (Camilo-Camargo) [inline src/modules/auth/templates/signup.tsx:1]: podriamos utilizar los componentes que ya existen para esto, creo que existe uno para Input

File: src/modules/auth/templates/signup.tsx [hunk 2]
```diff
@@ -1,10 +1,76 @@
...
+          <form action={dispatch} className="flex flex-col gap-4 min-w-80 ">
+            <label htmlFor="email">Email</label>
+            <Input
+              label="Escribe tu email"
+              name="email"
+              required
+              type="email"
+              radius="sm"
+              variant="bordered"
+            />
```
Comment (CamiloArias12) [inline src/modules/auth/templates/signup.tsx:30]:
```suggestion
          <Input
                            label="Email"
                            name="email"
                            required
                            type="email"
                            labelPlacement="outside"
                            placeholder="Escribe tu email"
                            radius="sm"
                            classNames={
                                {
                                    inputWrapper: "h-[50px] ",
                                    label: 'pb-1'
                                }
                            }
                            variant="bordered"
                        />
```

**Review by CamiloArias12 — COMMENTED** [review]
.

File: src/modules/auth/templates/signup.tsx [hunk 3]
```diff
@@ -1,10 +1,83 @@
...
   return (
-    <section>
-      <div></div>
-      <div></div>
-    </section>
+    <div className="flex flex-col border justify-center items-center w-1/2 h-screen gap-5">
+      <span className="font-bold items-left text-3xl m-6">Optitech</span>
+      <section className="flex flex-col m-6">
+        <h1 className="font-bold text-3xl">Registrarse</h1>
+        <span className="text-lg">Registrate para usar optitech</span>
```
Comment (Alejandro-Suarez-Rincon) [inline src/modules/auth/templates/signup.tsx:19]: Tildes
```suggestion
        <span className="text-lg">Regístrate para usar optitech</span>
```

**Review by CamiloArias12 — APPROVED** [review]

**Review by Alejandro-Suarez-Rincon — APPROVED** [review]

**Pull request merged**